doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -211,6 +211,8 @@ nRF Desktop
   * Changed the term *flash memory* to *non-volatile memory* for generalization purposes.
   * The :ref:`nrf_desktop_usb_state` to use the :c:func:`usb_hid_set_proto_code` function to set the HID Boot Interface protocol code.
     The ``CONFIG_USB_HID_PROTOCOL_CODE`` Kconfig option is deprecated and a dedicated API needs to be used instead.
+  * The :ref:`nrf_desktop_watchdog` to use ``watchdog0`` DTS alias instead of ``wdt`` DTS nodelabel.
+    Using the alias makes module's configuration more flexible.
```

Comment:
```suggestion
    Using the alias makes the configuration of a module more flexible.
```
Are we talking about a certain module or any module?

Reply:
`of the watchdog module` -> aligned :)